packages/arch/packages/drawer/src/index.js

```diff
@@ -181,11 +183,13 @@ let ModalDialog = memo<Props>(function ModalDialog({
 
   return createPortal(
     <Fragment>
-      <Blanket
-        style={fade(transitionState)}
-        onClick={closeOnBlanketClick ? onClose : undefined}
-        isTinted
-      />
+      {isOpen ? (
+        <Blanket
+          style={fade(transitionState)}
+          onClick={closeOnBlanketClick ? onClose : undefined}
+          isTinted
+        />
+      ) : null}
```

Comment:
Without this check, Cypress would _sometimes_ complain about a button not being clickable because this element was still on the page.

The `<Blanket>` element is used to capture clicks outside the modal to then close that modal. When `isClosed === true`, that functionality is no longer needed, so this `<Blanket>` should be removed.